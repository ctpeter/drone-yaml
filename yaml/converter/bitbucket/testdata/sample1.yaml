pipelines:
  default:
    - step:
        name: Build and test
        image: node:8.5.0
        caches:
          - node
        script:
          - npm install
          - npm test
          - npm build
        artifacts:
          - dist/**
    - step:
        name: Integration test
        image: node:8.5.0
        caches:
          - node
        get_env_var_from_vault_sg_staging:
          - postgres
        script:
          - npm run integration-test
    - step:
        name: Deploy to beanstalk
        image: python:3.5.1
        script:
          - python deploy-to-beanstalk.py
 
definitions:
  get_env_var_from_vault_sg_staging:
    postgres:
      image: postgres:9.6.4
