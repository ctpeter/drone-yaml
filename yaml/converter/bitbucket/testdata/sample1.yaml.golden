---
kind: pipeline
name: default

platform:
  os: linux
  arch: amd64

steps:
- name: Build and test
  image: node:8.5.0
  commands:
  - npm install
  - npm test
  - npm build

- name: Integration test
  image: node:8.5.0
  commands:
  - npm run integration-test

- name: Deploy to beanstalk
  image: python:3.5.1
  commands:
  - python deploy-to-beanstalk.py

get_env_var_from_vault_sg_staging:
- name: postgres
  image: postgres:9.6.4

...
